csharp/ql/lib/semmle/code/csharp/exprs/Call.qll

```diff
@@ -182,12 +182,20 @@ class Call extends DotNet::Call, Expr, @call {
   /**
    * Gets the argument that corresponds to parameter `p` of a potential
    * run-time target of this call.
+   *
+   * Does not consider default arguments.
```

Comment:
Does this predicate consider named arguments? (This question is not related to delegates.) It might be worth doing this change in a separate PR.

Reply:
We could change the comment from `Does not consider default arguments.` to 
```
   * Does not consider 
   *   - default arguments,
   *   - named arguments.
```